src/test/java/core/basesyntax/service/operation/PurchaseHandlerImplTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.service.operation;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitRecord;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class PurchaseHandlerImplTest {
+    private static OperationHandler operationHandler;
+    private Map<String, Integer> expected;
+    private FruitRecord fruitRecord;
+
+    @BeforeAll
+    static void setUp() {
+        operationHandler = new PurchaseHandlerImpl();
+    }
+
+    @Test
+    void checkFruitPurchase_Ok() {
+        expected = new HashMap<>();
+        expected.put("banana",10);
+        Storage.fruitMap.put("banana",20);
+        fruitRecord = new FruitRecord("b","banana",10);

```

Comment:
Is "**b**" the correct type of operation for Purchase?

Reply:
you are right. changed